### PR TITLE
Add .hidden to TileGrid and Group

### LINKED ISSUE #1688: displayio - allow ability to hide/show groups
a flag property called `visible` or something, to quickly hide/show a group would be great, easier than popping/pushing when you have a big group stack?

### PR DESCRIPTION
**Pull request by tannewt**
This allows for one to preserve ordering within a Group while
hiding something temporarily.

Fixes #1688

### CODE REVIEW

**Comment by ladyada** [comment]
yayy

File: shared-module/displayio/TileGrid.h [hunk 1]
```diff
@@ -57,15 +57,18 @@ typedef struct {
     displayio_area_t current_area; // Stored as an absolute area so it applies across frames.
     bool partial_change;
```
Comment (dhalbert) [inline shared-module/displayio/TileGrid.h]: Is it worth storing these as one-bit booleans to save space in the long run?

File: shared-module/displayio/Group.h [hunk 2]
```diff
@@ -49,11 +49,14 @@ typedef struct {
     uint16_t max_size;
     bool item_removed;
```
Comment (dhalbert) [inline shared-module/displayio/Group.h]: Store as one-bit booleans?

**Comment by tannewt** [comment]
@dhalbert Done and done.

**Comment by tannewt** [comment]
Ok, @dhalbert this passes all of the checks now.

**Pull request merged**